Nos/Views/NoteComposer/ImagePickerUIViewController.swift

```diff
@@ -15,6 +15,7 @@ struct ImagePickerUIViewController: UIViewControllerRepresentable {
         imagePicker.sourceType = sourceType
         imagePicker.mediaTypes = [UTType.image.identifier, UTType.movie.identifier]
         imagePicker.delegate = context.coordinator
+        imagePicker.allowsEditing = true
```

Comment:
One annoying thing this seems to have done is added a square crop box on the confirmation screen. It makes me thing that it's going to crop my image to square, but if I proceed it actually uploads the full image. The square is actually nice for profile photo upload though... can you see if there's a way to disable the crop box outside of the profile image flow? The confirmation screen is nice but if we have to remove that too I think it's better to have no confirmation screen than a misleading one.

<img src="https://github.com/user-attachments/assets/2ff62515-fe41-4eff-97d0-d558c861dfb3" width="300" />

Reply:
Wait, you can't even drag around the image to crop it. It just snaps back. Come on, Apple...

https://github.com/user-attachments/assets/04434a3a-0dec-4bf7-a829-d666d2efc74e

Reply:
Yeah, this is a bummer. Initially I got the image cropping to work: you just have to ask the `UIImagePickerController` for the `.editedImage` key rather than the `.originalImage` key, but I experienced the same problem you did here where it keeps snapping back. So in the end I removed the ability to edit the image after selection. I wonder if Apple has abandoned `UIImagePickerController` without actually deprecating it. The `photoLibrary` source type is deprecated in favor of `PHPickerViewController`.  

<img width="601" alt="Screenshot_8_29_24__8_19 AM" src="https://github.com/user-attachments/assets/727c9d9c-861a-4dd8-9664-2307ebab35b0">